src/main/java/nl/tudelft/scrumbledore/Constants.java

```diff
@@ -56,7 +56,7 @@ private Constants() {
   public static final String LEVELLABEL = "Level:";
   
   // Defining labels and values needed for the advancement GUI pop-ups.
-  public static final String ADVANCINGLABEL = "Advancing to next level in 5 seconds...";
+  public static final String ADVANCINGLABEL = "Advancing to next level in a few moments...";
```

Comment:
What about updating the seconds till advancing realtime? Since this message is drawn at each GUI refresh cycle, you can just put the remaining number of miliseconds there right?
<feature